db/schema.rb

```diff
@@ -161,7 +161,7 @@
     t.string "magic_link_token"
     t.datetime "magic_link_token_sent_at"
     t.boolean "hide_in_reporting", default: false, null: false
-    t.integer "course_from_find_id"
+    t.bigint "course_from_find_id"
```

Comment:
This shouldn't break anything right?

Reply:
Consensus is not, though we don't completely understand it. See also https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1263#discussion_r374214067